### ISSUE TITLE
Add saving sketches

### ISSUE THREAD

**Issue by Nircek** [Nircek/resistorer#3]
To do this there must be no `addr` or `parent ` in `element`.

**Comment by Nircek** [comment]
Pickle is good:
```
Python 3.7.1 (default, Oct 22 2018, 10:41:28) 
[GCC 8.2.1 20180831] on linux
Type "help", "copyright", "credits" or "license" for more information.
(InteractiveConsole)
>>> import pickle
>>> pickle.dumps(self)
Traceback (most recent call last):
  File "<console>", line 1, in <module>
TypeError: can't pickle _tkinter.tkapp objects
```

**Comment by Nircek** [comment]
It can be done by deleting `board.tk` and `board.w` then dumping.
And loading, replacing `board.tk` and `board.w`.

**Comment by Nircek** [comment]
Implemented in 27c2bf9, 59eff0c, e41738d and ebd49ee.
